fs/globImport.ts

```diff
@@ -63,7 +63,8 @@ export type GlobImportOptions = {
 function makeImportFunction(
   filePath: string,
   handlers: Record<string, ImportFactory>,
-) {
+  // deno-lint-ignore no-explicit-any
+): () => Promise<any> {
```

Comment:
Might be good to extract a helper type (`ImportLike`?) to reduce the number of lint ignore comments.